recipes/s3fs-fuse/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+./autogen.sh
+
+export CXXFLAGS="-I${PREFIX}/include -I${PREFIX}/include/libxml2"
+export LDFLAGS="-L${PREFIX}/lib"
```

Comment:
```suggestion
```
because `-I${PREFIX}/include` and `-L${PREFIX}/lib` are always included in our compiler/linker flags (plus, if you overwrite rather than append to the flags, you (likely unintentionally) drop flags we use for compatibility reasons).
We also don't need `-I${PREFIX}/include/libxml2` because `s3fs-fuse`'s `configure.ac` uses `PKG_CHECK_MODULES` which takes care of this for us.

Reply:
thanks  @mbargull , you're right, I forgot to append the current flag. but as you recommended I just removed these lines. thanks for the review.